doc/DataPlaneCodeGeneration/DeveloperDrivenEvolution.md

```diff
@@ -97,7 +97,28 @@ public virtual async Task<Response<MetricFeedback>> GetMetricFeedbackValuesAsync
 If the suggested rename does not apply to your scenario, you could pick the best name fitting your API.
 
 ## Implement the method
-You could call the protocol method inside your convenience method and map the returned raw response to model. See different scenarios and corresponding examples below. 
+You could call the protocol method inside your convenience method and map the returned raw response to model.
+
+In the implementation, you should only have one `DiagnosticScope` inside instead of creating `DiagnosticScope` for both protocol method and convenience method. Reasons are:
```

Comment:
Let's change this to something like:

The protocol method opens a diagnostic scope for the operation.  This can be used by the convenience method in many cases.   Only open a `DiagnosticScope` if:
- The convenience method has a different name than the protocol method.
- There is significant logic in the convenience method beyond model serialization/deserialization.
- If you need to add attributes or otherwise modify the diagnostic scope.

If a diagnostic scope is added in the convenience method, it should be suppressed in the inner protocol method so that there is only one distributed tracing span corresponding to the public method invocation.  This should happen by default in DPG clients, where `suppressNestedClientActivities` is set to true in the `ClientDiagnostics` constructor.